scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -577,6 +577,16 @@ SG_ DUMMY_AIR_SHUTDOWN : 0|1@1+ (1,0) [0|1] "" DCM
 BO_ 510 DIM_MOTOR_SHUTDOWN_ERRORS: 8 DIM
 SG_ DUMMY_MOTOR_SHUTDOWN : 0|1@1+ (1,0) [0|1] "" DCM
 
+BO_ 600 CLEAR_INV_FAULT_CMD: 1 DEBUG
+SG_ Clear_Left_Fault : 0|1@1+ (1,0) [0|1] "" DCM
+SG_ Clear_Right_Fault : 1|1@1+ (1,0) [0|1] "" DCM
+
+BO_ 601 INV_LOAD_SWITCHES_CMD: 1 DEBUG
+SG_ Close_Left_Switch : 0|1@1+ (1,0) [0|1] "" DCM
+SG_ Close_Right_Switch : 1|1@1+ (1,0) [0|1] "" DCM
+SG_ Open_Left_Switch : 2|1@1+ (1,0) [0|1] "" DCM
+SG_ Open_Right_Switch : 3|1@1+ (1,0) [0|1] "" DCM
```

Comment:
shouldnt this be two switches, left and right, with 0 being open 1 being closed?

Reply:
yep, not sure why I did that

Reply:
done